.dotnet/tests/Assistants/AssistantTests.cs

```diff
@@ -351,25 +353,24 @@ public void SettingResponseFormatWorks()
         AssistantClient client = GetTestClient();
         Assistant assistant = client.CreateAssistant("gpt-4o-mini", new()
         {
-            ResponseFormat = AssistantResponseFormat.JsonObject,
+            ResponseFormat = AssistantResponseFormat.CreateAutoFormat(),
         });
         Validate(assistant);
-        Assert.That(assistant.ResponseFormat, Is.EqualTo(AssistantResponseFormat.JsonObject));
+        Assert.That(assistant.ResponseFormat == "auto");
         assistant = client.ModifyAssistant(assistant, new()
         {
-            ResponseFormat = AssistantResponseFormat.Text,
+            ResponseFormat = AssistantResponseFormat.CreateTextFormat(),
         });
-        Assert.That(assistant.ResponseFormat, Is.EqualTo(AssistantResponseFormat.Text));
+        Assert.That(assistant.ResponseFormat == AssistantResponseFormat.CreateTextFormat());
```

Comment:
If you implement `IEquatable`, can you do `Is.EqualTo` again?

Reply:
Yep, great idea. Added that (+tests) *additionally* to these existing ones for the operators.